pack/valk_encounter.json

```
@@ -9,13 +9,15 @@
 		"position": 28,
 		"quantity": 1,
 		"set_code": "valk",
+		"text": "<b>Give to the Brunnhilde player.</b>\nValkyrie cannot attack the enemy with Death-Glow attached.\n<b>Alter-ego Action</b>: Spend [energy][mental] resources -> remove Trouble in Otherworld from the game.",
```

According to most commonly used format of the sentence _"Give to ..."_ it seems it should be both bold and italic.

```suggestion
		"text": "<b><i>Give to the Brunnhilde player.</i></b>\nValkyrie cannot attack the enemy with Death-Glow attached.\n<b>Alter-Ego Action</b>: Spend [energy][mental] resources → remove Trouble in Otherworld from the game.",
```